q2_types/ordination/_transformer.py

```diff
@@ -7,11 +7,26 @@
 # ----------------------------------------------------------------------------
 
 import skbio
+import pandas as pd
+import qiime2
 
 from ..plugin_setup import plugin
 from . import OrdinationFormat
 
 
+def _ordination_format_to_ordination_results(ff):
+    return skbio.OrdinationResults.read(str(ff), format='ordination',
+                                        verify=False)
+
+
+def _ordination_format_to_dataframe(ff):
+    ordination_results = _ordination_format_to_ordination_results(ff)
+    df = ordination_results.samples
+    df.index.name = 'ID'
+    df.columns = ['Column %d' % i for i in range(1, len(df.columns) + 1)]
```

Comment:
Can you change the column names to "Axis 1", "Axis 2", etc.? We don't have much info to work from given that the file format (and `OrdinationResults` object) are lossy, but "Axis" is a little more informative and may hint to the user that they're looking at axis coordinates.